src/kube-runtime/src/init

```diff
@@ -78,6 +86,17 @@ PAI_RUNTIME_DIR=${PAI_WORK_DIR}/runtime.d
 
 PAI_LOG_DIR=${PAI_WORK_DIR}/logs/${FC_POD_UID}
 
+# Move previous logs to another folder. Notice: for init.log, some part will append to previous log file
+LOG_FILES=$(find $PAI_LOG_DIR -maxdepth 1 -type f)
+if [[ ! -z $LOG_FILES ]]; then
+  PRE_LOG_DIR=$PAI_LOG_DIR/prelog_$(date +%s)
+  mkdir $PRE_LOG_DIR
+  mv $LOG_FILES $PRE_LOG_DIR
+fi
+
+# Clean ${PAI_WORK_DIR} since it may contain last execution content. (rarely happen, but seen in real world)
+rm -rf ${PAI_WORK_DIR}/*
```

Comment:
use a different folder each time: Also for PAI_WORK_DIR

Reply:
Hmm, this folder is created by and mounted by k8s: https://github.com/microsoft/pai/blob/2a2a6e25cca3bc2b272e2faf88d000d6919b9a7b/src/rest-server/src/models/v2/job/k8s.js#L318
The folder name is fixed during pod creation. We can't assign arbitrary name for different container.

And this folder content will no be persisted and always be the PAI runtime source code. (Also this folder will be deleted when deleting pod). So I think it's not necessary to use the different folder each time